### PR TITLE
chore: Change authorship reposense

### LINKED ISSUE #143: chore: Change authorship reposense

### PR DESCRIPTION
**Pull request by wenenhoe**
# Checklist for feature implementation and modification
Closes #143
- [x] Added relevant milestone (under milestone tab on the right)

### CODE REVIEW

**Review by nickczh — CHANGES_REQUESTED** [review]
Thanks Wen En for the effort to change the authorship repo-sense!

https://nus-cs2113-ay2324s2.github.io/website/admin/tools.html#tool-reposense-for-authorship-tracking

In the link above, the indication is usually:
`//@@author johndoe`
without a space after the backslash.

Not sure if that is necessary, but for your consideration. Additionally, an empty author tag is only when you don't know who wrote the code segment below. Some empty author tags already don't have more code below them, so may not be necessary?

**Comment by wenenhoe** [comment]
According to the [RepoSense docs](https://reposense.org/ug/usingAuthorTags.html), it is part of the supported comment formats. Additionally, the docs also mention that the empty author tag is used to denote the end of a code segment written by the author.

File: src/main/java/meditracker/command/AddCommand.java [hunk 1]
```diff
@@ -162,3 +163,4 @@ private void assertionTest() {
     }
 
 }
+// @@author nickczh
```
[the remark this answers is not in the document]
Comment (wenenhoe) [reply src/main/java/meditracker/command/AddCommand.java]: Good catch! Resolved in 82b084f

**Review by T0nyLin — APPROVED** [review]
Thank you for the changes in authorship!

**Review by kyuichyi — APPROVED** [review]
LGTM! Thanks for the changes!

**Review by nickczh — APPROVED** [review]
LGTM! Thank you for the changes.

**Pull request merged**